api/nakadi-event-bus-api.yaml

```diff
@@ -170,6 +170,10 @@ paths:
           description: Server error
           schema:
             $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporary) not available
```

Comment:
Seems you are right, "temporarily" is the adverb of course, "temporary" the adjective.